### PR TITLE
Disable trailingComma for Angular internal parser

### PR DESCRIPTION
**Pull request by fisker**
extra comma, cause parse error, fond in #6910

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-js/printer-estree.js [hunk 1]
```diff
@@ -120,6 +120,11 @@ const {
 let uid = 0;
 
 function shouldPrintComma(options, level) {
+  // angular does not allow trailing comma
+  if (options.parser === "__ng_interpolation") {
```
Comment (fisker) [inline src/language-js/printer-estree.js]: I'm not sure about `__ng_action`, `__ng_binding` , `__ng_directive`, not using angular

Comment (thorn0) [reply src/language-js/printer-estree.js]: Trailing comma should be disabled for all of them.

Comment (fisker) [reply src/language-js/printer-estree.js]: you are right, done

**Comment by fisker** [comment]
we need this for #6910

File: src/language-js/printer-estree.js [hunk 2]
```diff
@@ -120,6 +120,16 @@ const {
 let uid = 0;
 
 function shouldPrintComma(options, level) {
+  // angular does not allow trailing comma
+  if (
+    options.parser === "__ng_interpolation" ||
+    options.parser === "__ng_action" ||
+    options.parser === "__ng_binding" ||
+    options.parser === "__ng_directive"
+  ) {
```
Comment (evilebottnawi) [inline src/language-js/printer-estree.js]: Can we simplify this using regexp `^__ng_`?

Comment (fisker) [reply src/language-js/printer-estree.js]: really nessary? only 4, but I was thinking override here, https://github.com/prettier/prettier/blob/4ed525350904a3c19f4a7504e53a27a98b12531e/src/language-js/parser-angular.js#L5 what do you suggest?

Comment (fisker) [reply src/language-js/printer-estree.js]: Just realize this options is not passing to print function

```js
options.trailingComma = "none";
```

seems not safe. maybe I should keep check in shouldPrintComma

Comment (fisker) [reply src/language-js/printer-estree.js]: @evilebottnawi still prefer regexp?

Comment (evilebottnawi) [reply src/language-js/printer-estree.js]: It is just question, i am not familiar with angular very good,all fine

**Pull request merged**